app/commands/decidim/time_tracker/admin/export_time_tracker.rb

```diff
@@ -89,9 +89,7 @@ def export_time_tracker
               @user,
               params_category
             )
-
-            status = @not_started
-            status = @work_in_progress unless task.activities.joins(:time_events).empty?
+            @work_in_progress unless task.activities.joins(:time_events).empty? # rubocop:disable Lint/Void
```

Comment:
I don't get what this line does. Shouldn't be `status = @work_in_progress`. Otherwise, this line is useless and could be removed